plugin/core/transports.py

```diff
@@ -95,23 +95,25 @@ def _decode(message: bytes) -> Dict[str, Any]:
 class ProcessTransport(Transport[T]):
 
     def __init__(self, name: str, process: subprocess.Popen, socket: Optional[socket.socket], reader: IO[bytes],
-                 writer: IO[bytes], stderr: Optional[IO[bytes]], processor: AbstractProcessor[T],
+                 writer: IO[bytes], processor: AbstractProcessor[T],
```

Comment:
We shouldn't change this signature because this class used by [external package](https://github.com/sublimelsp/LSP-file-watcher-chokidar) and so it would be a breaking change.

We can just pass `None` since it was already optional.